### ISSUE TITLE
Image upload for maps fails the first time

### ISSUE THREAD

**Issue by Kordishal** [worldanvil/worldanvil-bug-tracker#1106]
**World Anvil Username**: SoulLink
**Feature**: Chronicles

**Describe the Issue**
For me consistently when I create a new map it fails the first time around. The fail shows via a red alert message. However then it does not actually stop trying to create the map and just loads forever until I hit the small X to close the map creation form. Then I start again add all the data and add the image and the second time it has always worked.

This happened to me three out of three times I uploaded a WEBP map (sized way bellow 10 MB). 

**Expected behavior**
When the upload fails it should actually stop and return to the map creation form. So that I don't have to reinsert the title etc. and can try with other images.

And of course it would be good if it worked the first time ;)

GitHub unfortunately doesn't support WEBP, but I can send them to you via DM.

**Comment by GinaCaeli** [comment]
I am having this problem as well, but it persistant under certain circumstances. I have tried to upload a new map image to an existing Chronicle (a PNG file 5 or 6 times, and a WEBP twice - on Chrome).  All of those attempts failed.
I then tried to create a completely new chronicle and my map image failed to upload in both formats.  The cover image (a jpg) wass fine, so I tried a png as the cover image instead, and it also failed.  
After that, I switched to a JPG for the map on an existing chronicle and was able to replicate the bug above, as Kordishal/SoulLink described it.
Lastly, I tried starting another new chronicle, and the jpg map loaded on the first time.
I hope this info helps in some way.
